bin/hdfcoinc/pycbc_stat_dtphase2

```diff
@@ -0,0 +1,149 @@
+#!/bin/env python
+""" Create a file containing the phase and amplitude, 
+correlations between two detectors by
+doing a simple monte-carlo
+"""
+import argparse, h5py, numpy.random, pycbc.detector, logging, multiprocessing
+from numpy.random import normal, uniform, power
+from scipy.stats import norm
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--ifos', nargs='+',
+                    help="The two ifos to generate a histogram for")
+parser.add_argument('--sample-size', type=int, 
+                    help="Approximant number of independent samples to draw for the distribution")
+parser.add_argument('--snr-ratio', type=float, 
+                    help="SNR ratio permitted between reference ifo and all others")
+parser.add_argument('--relative-sensitivities', nargs='+', type=float)
+parser.add_argument('--seed', type=int, default=124)
+parser.add_argument('--output-file')
+parser.add_argument('--verbose', action='store_true')
+parser.add_argument('--smoothing-sigma', type=int, default=2)
+args = parser.parse_args()
+
+# Apply a simple smoothing to help account for
+# measurement errors for weak signals
+def smooth_param(data, sigma, index):
+    bins = numpy.arange(-args.smoothing_sigma, args.smoothing_sigma+1)
+    kernel = norm.pdf(bins)
+    
+    nweights = {}
+    
+    # This is massively redundant as many points may be 
+    # recalculated, but it's straightforward.
+    for key in weights:
+        for a in bins:
+            nkey = list(key)
+            nkey[index] += a
+            
+            weight = 0
+            for b, w in zip(bins, kernel):
+                wkey = list(nkey)
+                wkey[index] += b
+                wkey = tuple(wkey)
+                
+                if wkey in data:
+                    weight += data[wkey] * w
+                
+            nweights[tuple(nkey)] = weight
+            
+    return nweights
+    
+d = {ifo: pycbc.detector.Detector(ifo) for ifo in args.ifos}
+
+pycbc.init_logging(args.verbose)
+
+numpy.random.seed(args.seed)
+size = 1000000
+chunks = int(args.sample_size / size) + 1
+
+l = 0
+nsamples = 0
+weights = {}
+for k in range(chunks):
+    nsamples += size
+    logging.info('generating %s samples' % size)
+
+    # Choose random sky location and polarizations from
+    # an isotropic population
+    ra = uniform(0, 2 * numpy.pi, size=size)
+    dec = numpy.arccos(uniform(-1., 1., size=size)) - numpy.pi/2
+    inc = numpy.arccos(uniform(-1., 1., size=size))
+    pol = uniform(0, 2 * numpy.pi, size=size)
+    ip = numpy.cos(inc)
+    ic = 0.5 * (1.0 + ip * ip)
+
+    twidth = .001  # approximate timing error at lower SNRs
+    pwidth = .25   # approximate phase error at lower SNRs
+    serr = 1.4   #
+    sref = 5.0     # Reference SNR for bin smoothing
```

Comment:
Are these values constant enough throughout different cases to not require command-line input?